src/meta.jl

```diff
@@ -20,10 +20,10 @@ macro op(f)
         end => opname
     end
     opname === nothing && error("Invalid usage of @op")
-    # opname = f.args[1].args[1]
+
     already_registered = opname ∈ registered_ops
     push!(registered_ops, opname)
-    @assert(isdefined(:tf)) # Need tf as name for module where this code is located
```

Comment:
Ah ya, it raises a deprecation warning on .7 though. 

Instead of just addressing the deprecation though, can we think of a more elegant solution than requiring that user code has a certain symbol defined>